indra/newview/llvovolume.cpp

```diff
@@ -998,6 +1001,28 @@ LLDrawable *LLVOVolume::createDrawable(LLPipeline *pipeline)
     return mDrawable;
 }
 
+// Inverse of gMeshRepo.loadMesh and gMeshRepo.getSkinInfo, combined into one function
+// Assume a Collada mesh never changes after being set.
```

Comment:
This is a big assumption. If the sculpt ID were to ever change, we would also need to call `unregisterOldMeshAndSkin` from `LLPrimitive::setVolume`.

It's my understanding that features to change sculpt IDs were disabled long ago.